storage/connectors/jdbc/src/test/java/feast/storage/connectors/jdbc/retriever/snowflake_entity_rows_test1.csv

```diff
@@ -0,0 +1,4 @@
+entity_id_primary,created_timestamp
+1,2019-12-31 17:00:00
```

Comment:
Ideally these types of files would either be named descriptively based upon the scenario or data condition they contain

Reply:
they also aren't snowflake specific, so that part of the name is not necessary.  And rows is redundant. How about something like `entities_2dates.csv` or something?

Reply:
ok, let me update with a better name.